pkg/variantregistry/ocp.go

```diff
@@ -838,6 +842,7 @@ func setPlatform(jLog logrus.FieldLogger, variants map[string]string, jobName st
 		{"-rosa", "rosa"}, // Keep above AWS as many ROSA jobs also mention AWS
 		{"-aws", "aws"},
 		{"-alibaba", "alibaba"},
+		{"-azure-aro-hcp", "aro"},
```

Comment:
I couldn't see that any variants other than owner, installer, platform made sense for `aro-integration`. maybe LayeredProduct? well, we can always adjust later.